tests/acceptance/01_vars/01_basic/CFE-628.cf

```diff
@@ -0,0 +1,39 @@
+###########################################################
+#
+# Test that a variable reference with nested parentheses works
+#
+###########################################################
+
+body common control
+{
+    inputs => { "../../default.cf.sub" };
+    bundlesequence => { default($(this.promise_filename)) };
+    version => "1.0";
+}
+
+###########################################################
+
+bundle agent test
+{
+  meta:
```

Comment:
Mark as !windows only test. Since you are using `/bin/echo`.

Reply:
ACK,